dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/netty/jfapchannel/JMSClientInboundHandler.java

```diff
@@ -0,0 +1,127 @@
+package com.ibm.ws.netty.jfapchannel;
```

Comment:
Missing copyright headers in this package

Reply:
Just added, thanks!